src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,160 @@
+        // given
+        String lineName1 = "신분당선";
+        Long upStationId1 = createStationByStationName("강남역").jsonPath().getLong("id");
+        Long downStationId1 = createStationByStationName("신논현역").jsonPath().getLong("id");
+        Map<String, Object> requestBody = createRequestBody(lineName1, "bg-red-600", upStationId1, downStationId1, 10);
+
+        createLine(requestBody);
+
+        String lineName2 = "수인분당선";
+        Long upStationId2 = createStationByStationName("압구정로데오역").jsonPath().getLong("id");
+        Long downStationId2 = createStationByStationName("강남구청역").jsonPath().getLong("id");
+        Map<String, Object> requestBody2 = createRequestBody(lineName2, "bg-yellow-600", upStationId2, downStationId2, 10);
+
+        createLine(requestBody2);
+
+        // when & then
+        List<String> lineNames =
+                selectLines().jsonPath().getList("name", String.class);
+        assertThat(lineNames).containsAnyOf("신분당선", "수인분당선");
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 조회하면
+     * Then 생성한 지하철 노선의 정보를 응답받을 수 있다.
+     */
+    @DisplayName("지하철 노선을 조회한다.")
+    @Test
+    void 지하철_노선을_조회() {
+        // given
+        String lineName = "신분당선";
+        Long upStationId = createStationByStationName("강남역").jsonPath().getLong("id");
+        Long downStationId = createStationByStationName("신논현역").jsonPath().getLong("id");
+        Map<String, Object> requestBody = createRequestBody(lineName, "bg-red-600", upStationId, downStationId, 10);
+
+        Long lineId = createLine(requestBody).jsonPath().getLong("id");
+
+        // when & then
+        String responseLineName = selectLine(lineId, HttpStatus.OK).jsonPath().get("name");
+        assertThat(lineName).isEqualTo(responseLineName);
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 수정하면
+     * Then 해당 지하철 노선 정보는 수정된다.
+     */
+    @DisplayName("지하철 노선을 수정한다.")
+    @Test
+    void 지하철_노선을_수정() {
+        // given
+        String lineName = "신분당선";
+        Long upStationId = createStationByStationName("강남역").jsonPath().getLong("id");
+        Long downStationId = createStationByStationName("신논현역").jsonPath().getLong("id");
+        Map<String, Object> requestBody = createRequestBody(lineName, "bg-red-600", upStationId, downStationId, 10);
+
+        Long lineId = createLine(requestBody).jsonPath().getLong("id");
+
+        // when
+        Long newDownStationId = createStationByStationName("양재역").jsonPath().getLong("id");
+        String newLineName = "구분당선";
+        Map<String, Object> modifyRequestBody = createRequestBody(newLineName, "bg-red-600", upStationId, newDownStationId, 10);
+
+        modifyLine(lineId, modifyRequestBody);
+
+        // then
+        JsonPath responseJsonPath = selectLine(lineId, HttpStatus.OK).jsonPath();
+        List<String> stationNames = responseJsonPath.getList("stations.name");
+
+        assertThat(newLineName).isEqualTo(responseJsonPath.get("name"));
```

Comment:
순서를 반대로 바꿔보면 어떨까요? 우리가 검증하고자 하는 것이 assertThat 내부에, 그리고 검증의 결과값을 isEqualTo 내부로요.
```suggestion
        assertThat(responseJsonPath.get("name")).isEqualTo(newLineName);
```